tests/ui/lint/lint-unnecessary-refs.rs

```diff
@@ -0,0 +1,31 @@
+//@ run-rustfix
+
+#![deny(unnecessary_refs)]
```

Comment:
I would prefer that we do not override the lint level
```suggestion
```
and use `//@ check-pass`/`WARN`